docs/en/user_guides/visualization.md

```diff
@@ -1 +1,34 @@
-# visualization 
\ No newline at end of file
+# Visualization
+
+Using `visualizer` in config file can save visual results when training or testing.
+MMEditing supports `Loacl` visual backend and `Tensorboard ` visual backend now.
+
+## LocalVisBackend
+
+Here are configs using `LocalVisBackend` to save visual results as an example.
```

Comment:
Here, you may briefly tell users what are passed to visualizer, and they should set which fields to save visual results.